frontend/src/pages/Projects/Project/ChangeRequest/FormConfig.ts

```diff
@@ -51,11 +68,12 @@ export const FormConfig = (query: UseQueryResult<AxiosResponse, unknown>) => {
       required: true,
     },
     {
+      //TODO: Broken, wil be updated in future ticket
       fieldName: "cr_contact",
       fieldType: "singleText",
       fieldLabel: "CR Contact",
       width: "half",
-      required: true,
+      // required: true,
```

Comment:
is this comment suppose to be here?

Reply:
i'll remove the comment in my current PR and keep the TODO intact